src/modules/categories/components/CategoryList.js

```diff
@@ -18,17 +20,27 @@ type PropsType = {|
   query?: string,
   theme: ThemeType,
   onItemPress: (tile: { title: string, thumbnail: string, path: string }) => void,
-  language: string
+  language: string,
+  thumbnail?: string
 |}
 
+const CategoryThumbnail = styled(Image)`
+  align-self: center;
+  flex-shrink: 0;
+  width: 70px;
+  height: 70px;
+  margin: 10px;
+`
+
 /**
  * Displays a ContentList which is a list of categories, a caption and a thumbnail
  */
 class CategoryList extends React.Component<PropsType> {
   render () {
-    const { categories, title, content, query, theme, onItemPress, language } = this.props
+    const { categories, title, content, query, theme, onItemPress, language, thumbnail } = this.props
     return <>
-      {title && <Caption title={title} theme={theme} />}
+      {thumbnail && <CategoryThumbnail source={thumbnail} />}
+      {title && <CategoryListCaption title={title} theme={theme} withThumbnail={Boolean(thumbnail)} />}
```

Comment:
We usually use `!!thumbnail` as a shorthand for `Boolean(thumbnail)`. (you don't have to adjust this.)
```suggestion
      {title && <CategoryListCaption title={title} theme={theme} withThumbnail={!!thumbnail} />}
```